.github/workflows/release-reproducible-smart-contract.yaml

```
@@ -47,5 +46,3 @@ jobs:
           push: true
           platforms: linux/amd64
           tags: ${{ steps.meta.outputs.tag }}
-          build-args: |
-            source_image=${{ env.SOURCE_IMAGE_TAG }}
```

The idea is that we ensure that the docker `tag` matches the rust version inside of the docker build. Currently, we not ensuring this anymore. 

Meaning if I make a realize with the below CI pipeline tag, then the rust version in the docker images should be `1.87.0` and not the hardcoded one `1.85`

```
docker pull concordium/verifiable-sc:1.87.0
```

---

Yes, I think some validation was lost by the changes. I look primarily at the changes since Emils review, so did not see it, but yes. Seems like now the CI does not verify the tag matches the Rust version.

---

Though the release build will likely fail hard right now, since the validation code is still there :)